### PR TITLE
[DNM] Compliance clang-format test

### PR DESCRIPTION
**Pull request by pdgendt**
A test to validate #75089 works

### CODE REVIEW

File: samples/basic/blinky/src/main.c [hunk 1]
```diff
@@ -25,14 +25,12 @@ int main(void)
...
 	bool led_state = true;
 
-	if (!gpio_is_ready_dt(&led)) {
+	if (!gpio_is_ready_dt(&led))
 		return 0;
-	}
 
 	ret = gpio_pin_configure_dt(&led, GPIO_OUTPUT_ACTIVE);
-	if (ret < 0) {
+	if (ret < 0)
```
Comment (nashif) [inline samples/basic/blinky/src/main.c:32]: was it supposed to catch this? This is not allowed

Comment (nashif) [reply samples/basic/blinky/src/main.c:32]: this is confusing, I see that above.

Comment (pdgendt) [reply samples/basic/blinky/src/main.c:32]: This was done per request of @kartben here: https://github.com/zephyrproject-rtos/zephyr/pull/75089#pullrequestreview-2148174067

I don't have a strong preference, as long as the notice is automated by the script I'm fine.

Comment (pdgendt) [reply samples/basic/blinky/src/main.c:32]: @nashif as you can see the compliance check action actually succeeds, so the notations are merely suggestions, not blockers.

Comment (pdgendt) [reply samples/basic/blinky/src/main.c:32]: > was it supposed to catch this? This is not allowed

This isn't allowed, but there currently isn't any compliance check that would prevent this.

**Pull request closed without merging**